extendr-api/src/metadata.rs

```diff
@@ -46,6 +53,7 @@ pub struct Metadata {
 struct RArg {
     name: String,
     default: Option<&'static str>,
+    is_ellipsis: bool,
```

Comment:
Why is this modelled so that an `RArg` can *both* be ellipsis *and* have a default, whereas an `Arg` cannot be both? Can we be consistent here?

Reply:
I was lazy to properly handle this, but I can do that for sure!